components/user/UserCard.js

```diff
@@ -0,0 +1,43 @@
+import Link from "next/link";
+import ReactMarkdown from "react-markdown";
+import { abbreviateNumber } from "js-abbreviation-number";
+
+import FallbackImage from "../FallbackImage";
+
+export default function UserCard({ profile }) {
+  return (
+    <Link
+      href={`/${profile.username}`}
+      className="flex flex-col items-center border-2 max-w-[14rem] h-[17rem] overflow-hidden rounded-lg border-gray-200 h p-4 my-2 gap-3 "
+    >
+      <div className="">
+        <div className="flex justify-center">
+          <FallbackImage
+            src={profile.avatar}
+            alt={`Profile picture of ${profile.name}`}
+            width={80}
+            height={80}
+            className="rounded-full"
+            fallback={profile.name}
+          />
+        </div>
+        <h3 className="text-lg  md:hidden text-center mt-2 text-[#ff5a00] font-bold">
```

Comment:
why are the stats hidden on desktop? might make sense to hide for mobile as there is less space